test/solc/CommandLineParser.cpp

```diff
@@ -241,6 +241,13 @@ BOOST_AUTO_TEST_CASE(no_cbor_metadata)
 	BOOST_TEST(assert);
 }
 
+BOOST_AUTO_TEST_CASE(no_import_callback)
+{
+	vector<string> commandLine = {"solc", "--no-import-callback", "contract.sol"};
```

Comment:
I've added mutual exclusion for `--allow-paths` and `--no-import-callback`, so not sure how to pull this off, since the test uses `--allow-paths` (if you were referring to `cli_mode_options` test)?

Reply:
True, now you can't :) In that case I'd just loop over available input modes here.